frontend/src/helpers/utils.js

```diff
@@ -151,7 +151,8 @@ export const codesToDisplayText = {
         date_needed: "Need By Date",
         line_description: "Description",
         proc_shop_fee_percentage: "Shop Fee",
-        status: "Status"
+        status: "Status",
+        services_component: "Services Component",
```

Comment:
🤔 do we have a test we can update with this case?

Reply:
I'll can add a something to a unit test for this.
But if you have any ideas for e2e tests, let me know.  I wasn't if there was good place yet for that.

Reply:
We've added e2e tests now, too.